### PR TITLE
Remove pre-workshop branch notice

### PR DESCRIPTION
**Pull request by StefanoFioravanzo**
Removing pre KubeCon workshop notice that redirected users to the kubecon-workshop branch

### CODE REVIEW

**Review by elikatsis — COMMENTED** [review]
I'd say wait for next release, which I believe we could cut very soon [after 2-3 PRs that are almost ready and reviewed]. Otherwise, we could add a note that many features shown during the KubeCon workshop are not released yet (and remove it once we cut the release).

**Comment by StefanoFioravanzo** [comment]
Let's keep this pending then, we will merge after the next features are upstream as well

**Review by elikatsis — APPROVED** [review]

**Pull request merged**
